docs/testing.md

```
@@ -0,0 +1,105 @@
+xUnit + Moq, in `backend/unit-tests/Ahlcg.ApiService.Tests/`. Coverage via coverlet (`XPlat Code Coverage`), aggregated by `reportgenerator`.
+
+```bash
+cd backend
+dotnet test
+dotnet test --filter "FullyQualifiedName~AuthEndpointsTests"
+dotnet test --collect:"XPlat Code Coverage"
+```
+
+Test names follow `{Method}_{Scenario}_{Expected}`:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)